tasks/os-support-check.yml

```diff
@@ -2,7 +2,7 @@
 - name: Check if OS is supported
   fail:
     msg: "The SolarWinds Snap Agent is not supporting your OS"
-  when: ansible_os_family not in ["RedHat", "Debian", "Windows"]
+  when: ansible_os_family not in ["RedHat", "Debian", "Windows", "AlmaLinux"]
```

Comment:
It seems that assigning Alma Linux as "AlmaLinux" ansible os family was internal bug on Ansible side. https://bugs.almalinux.org/view.php?id=98 

It now shall be properly fixed on Ansible side (also for Rocky Linux): https://github.com/ansible/ansible/pulls?q=is%3Apr+almalinux+is%3Aclosed
```
[vagrant@localhost ~]$ cat /etc/*release
AlmaLinux release 8.4 (Electric Cheetah)
[...]
[vagrant@localhost ~]$ ansible localhost  -m setup | grep family
        "ansible_os_family": "RedHat",
```

May I ask you to double check it with newest version of ansible 2.9.x?  According to comment form BugTracker "After updating Ansible to 2.9.20 everything seems to be ok."